Makefile

```diff
@@ -507,3 +507,7 @@ generate-sqlx-data:  # Regenerate sqlx-data.json based on queries made in Rust c
 # Intentionally not phony, as this generates a real file
 dist/firecracker_kernel.tar.gz: firecracker/generate_firecracker_kernel.sh
 	./firecracker/generate_firecracker_kernel.sh
+
+FIRECRACKER_ROOTFS_DEPENDENCIES = $(shell find firecracker/rootfs)
```

Comment:
1. This should be := not =
2. Why do we need this find anyway?

Reply:
re: := / =, I was following the pattern of `ASSET_DIRS / ASSET_FILES` in the other makefile

The `find` is a way to list the directory and every file underneath it; we do this so that we don't needlessly rebuild the .tar.gz unless something in that directory has changed.

There's no great way to have make targets depend on a directory: https://stackoverflow.com/questions/4440500/depending-on-directories-in-make